db/models/index.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const fs        = require("fs");
+const path      = require("path");
+const Sequelize = require("sequelize");
+const basename  = path.basename(__filename);
+const env       = process.env.NODE_ENV || "development"; // eslint-disable-line no-process-env
+const config    = require(__dirname + "/../config/config.json")[env];
+const db        = {};
+
+let sequelize;
+
+if (config.use_env_constiable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config); // eslint-disable-line no-process-env
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
```

Comment:
I'm not sure if this would work since I don't see a `database`, `username`, or `password` field in [./db/config/config.json](https://github.com/mozilla/blurts-server/pull/67/files#diff-3050f5967be4a98d0035b27aa4050e97).